### PR TITLE
Promote APIs shipped in VS 2015 Update 1

### PR DESCRIPTION
**Pull request by amcasey**
Methodology: sync to 4cb203d, add any lines appearing in PublicAPI.Unshipped.txt to master's PublicAPI.Shipped.txt and remove them from master's PublicAPI.Unshipped.txt, re-sort.

p.s. Everything in 7de1e9f is a guess, so please review carefully.

### CODE REVIEW

**Comment by amcasey** [comment]
FYI @dotnet/roslyn, esp @jaredpar, @jasonmalinowski, @tmat

**Comment by amcasey** [comment]
Note that this PR is built on top of https://github.com/dotnet/roslyn/pull/8209.

**Comment by amcasey** [comment]
Does this resolve https://github.com/dotnet/roslyn/issues/7989?

**Comment by amcasey** [comment]
Ping, since notifications may have been lost while GH was down.

File: src/Compilers/Core/Portable/PublicAPI.Unshipped.txt [hunk 1]
```diff
@@ -65,27 +31,10 @@ Microsoft.CodeAnalysis.Diagnostics.OperationBlockStartAnalysisContext.OperationB
 Microsoft.CodeAnalysis.Diagnostics.OperationBlockStartAnalysisContext.Options.get -> Microsoft.CodeAnalysis.Diagnostics.AnalyzerOptions
 Microsoft.CodeAnalysis.Diagnostics.OperationBlockStartAnalysisContext.OwningSymbol.get -> Microsoft.CodeAnalysis.ISymbol
 Microsoft.CodeAnalysis.Diagnostics.OperationBlockStartAnalysisContext.RegisterOperationAction(System.Action<Microsoft.CodeAnalysis.Diagnostics.OperationAnalysisContext> action, params Microsoft.CodeAnalysis.Semantics.OperationKind[] operationKinds) -> void
-Microsoft.CodeAnalysis.Diagnostics.SuppressionInfo
-Microsoft.CodeAnalysis.Diagnostics.SuppressionInfo.Attribute.get -> Microsoft.CodeAnalysis.AttributeData
-Microsoft.CodeAnalysis.Diagnostics.SuppressionInfo.Id.get -> string
-Microsoft.CodeAnalysis.Diagnostics.Telemetry.AnalyzerTelemetryInfo
-Microsoft.CodeAnalysis.Diagnostics.Telemetry.AnalyzerTelemetryInfo.CodeBlockActionsCount.get -> int
```
Comment (CyrusNajmabadi) [inline src/Compilers/Core/Portable/PublicAPI.Unshipped.txt]: We're making telemetry stuff public?

**Comment by amcasey** [comment]
FYI, I'm planning to rewrite history shortly as I fix the merge conflict.

**Comment by amcasey** [comment]
Revised to reflect recent commits.

**Comment by jaredpar** [comment]
:+1:

**Comment by amcasey** [comment]
FYI @genlu, you might as well check in your change since there are already conflicts.

**Comment by amcasey** [comment]
This change can only affect compilation and at least one queue has compiled successfully.

**Pull request merged**